pcsx2-qt/Settings/AudioSettingsWidget.cpp

```diff
@@ -114,12 +114,18 @@ AudioSettingsWidget::AudioSettingsWidget(SettingsDialog* dialog, QWidget* parent
 	dialog->registerWidgetHelp(m_ui.syncMode, tr("Synchronization"), tr("TimeStretch (Recommended)"),
 		tr("When running outside of 100% speed, adjusts the tempo on audio instead of dropping frames. Produces much nicer fast-forward/slowdown audio."));
 
-	dialog->registerWidgetHelp(m_ui.expansionMode, tr("Expansion"), tr("Stereo (None, Default)"), tr(""));
+	dialog->registerWidgetHelp(m_ui.expansionMode, tr("Expansion"), tr("Stereo (None, Default)"), 
+		tr("Determines the speaker configuration to apply."));
```

Comment:
Perhaps it would be worthwhile being more specific here. Something along the lines of `Determines how the stereo output from the emulated system is upmixed into a greater number of speakers.`